### DISCUSSION TITLE
D2C9D07B-A865-4BF6-8A74-4045DEB34ED7

### DISCUSSION THREAD

**Discussion by giscus** [celestiamobile/comments#86] [General]
# Iota Horologii

Iota Horologii is a Sun-like star about 56.51 light-years away in the constellation Horologium. Appearing as a fifth-magnitude star as seen from Earth, it has a planet about 6.7 times more massive than Jupiter, completing one orbit once every 307 days. It is one of the first exoplanets ever discovered, having been first confirmed in 1999

https://celestia.mobi/addon?item=D2C9D07B-A865-4BF6-8A74-4045DEB34ED7&lang=en

**Comment by exocritic5457** [comment]
Better than that green Saturn.
